src/main/java/org/candlepin/audit/EventMessageReceiver.java

```diff
@@ -36,7 +36,7 @@ public EventMessageReceiver(EventListener listener, ActiveMQSessionFactory sessi
 
     @Override
     protected void initialize() throws Exception {
-        session = this.sessionFactory.getIngressSession(false);
+        session = this.sessionFactory.getIngressSession(true);
```

Comment:
This is a significant change that I want to highlight. I noticed that when a webhook failed it was not being requeued and therefore not retried as it should be. I found some code that gave me the impression that failed events _should_ be retried but in fact they are not. This might have happened when the implementation switched from qpid to artemis. Does anyone know the intention? The retry capability seems valuable. Maybe a bug?

https://github.com/candlepin/candlepin/blob/d1a51421118c5421788e876c8baa7e280e40527f/src/main/java/org/candlepin/audit/DefaultEventMessageReceiver.java#L80-L83

Reply:
fwiw, I do agree with your findings that those two bits of code seem at odds with each other. We have a ton of other things in front of it, but we probably should put something on the backlog to investigate that and, at the very least, remove the comments/dead code to avoid this confusion going forward.